client/src/model/apis/navigationApi/navigationApi.js

```diff
@@ -11,6 +11,7 @@ export const createNavigationApi = ({ store }) => {
     openTraining: '/trainings/:training',
     editTraining: '/trainings/:training/edit',
     createExercise: '/trainings/:training/new-exercise',
+    editExercise: '/trainings/:training/:exercise',
```

Comment:
```
editNewExercise: '/trainings/:training/:exercise/editNew',
editExistingExercise: '/trainings/:training/:exercise/edit',
```

Reply:
ok, you have really deleted `createExercise` flow by mistake (referring https://github.com/vincefliek/YourGym/pull/13#discussion_r905607870)